akka-cluster-bootstrap/src/main/resources/application.conf

```diff
@@ -0,0 +1,23 @@
+# TODO: What needs to happen to move this into rp.conf?
+
+akka {
```

Comment:
How is this application.conf supposed to be used? This is a library, right? Supposed to be included as a dependency in the application. A library can't include an application.conf in its jar. That would be a high risk classpath conflict/duplicate. A library can't override properties owned by another library in a reference.conf. That means that what you are trying to do is not possible and should instead be defined by the end user in their application.conf, possibly by including rp.conf.

In short, you can't and shouldn't try to make it automagic. Something must be added in the application's conf to enable this.

One exception is that Play has the special reference-overrides.conf, but that can only be used with Play and Lagom.

Reply:
I understand your concern, hence this part being marked TODO. What we'll probably end up doing is documenting that you include `rp.conf` in your `application.conf` file for regular apps. For Play and Lagom we can leverage `reference-overrides.conf`.